docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc

```diff
@@ -41,20 +43,22 @@ metadata:
 projects:
 - name: frontend
   git:
-    location: https://github.com/acmecorp/frontend.git
+    remotes:
+      origin: "https://github.com/acmecorp/frontend.git"
 - name: backend
   git:
-    location: https://github.com/acmecorp/backend.git
+    remotes:
+      origin: "https://github.com/acmecorp/backend.git"
 ----
 ====
 
 . For each project, define an unique value for the mandatory `name` attribute.
 
 
 
-. For each project, define a mandatory source of one of the following types: `git`, `github` and `zip`.
+. For each project, define a mandatory source of one of the following types: `git` or `zip`.
 
-`git`:: Projects with sources in Git. The location points to a clone link.
+`git`:: Projects with sources in Git. `checkoutFrom` points to a clone link.
```

Comment:
This information contradicts the example. The example demonstrates that `remotes` contains named remotes which actually point to Git clone URLs. `checkoutFrom` seems to specify the branch to be used.

Reply:
Cool, as you observed, I've changed it to state that "1checkoutFrom` points to the branch being used."